extensions/chat/client.mjs

```diff
@@ -18,17 +18,38 @@ class Client {
   }
 
   send_msg(client){
-    return function(evt){
+    return async function(evt){
```

Comment:
Note/Optional: If you use arrow function, `this` binding will not be overridden, and you can use this in your function instead of passing `client` in.